src/System.Globalization.Extensions/tests/GetStringComparerTests.cs

```diff
@@ -17,9 +17,9 @@ public void GetStringComparer_Invalid()
         {
             AssertExtensions.Throws<ArgumentNullException>("compareInfo", () => ((CompareInfo)null).GetStringComparer(CompareOptions.None));
 
-            AssertExtensions.Throws<ArgumentException>("options", () => new CultureInfo("tr-TR").CompareInfo.GetStringComparer((CompareOptions)0xFFFF));
-            AssertExtensions.Throws<ArgumentException>("options", () => new CultureInfo("tr-TR").CompareInfo.GetStringComparer(CompareOptions.Ordinal | CompareOptions.IgnoreCase));
-            AssertExtensions.Throws<ArgumentException>("options", () => new CultureInfo("tr-TR").CompareInfo.GetStringComparer(CompareOptions.OrdinalIgnoreCase | CompareOptions.IgnoreCase));
+            AssertExtensions.Throws<ArgumentException>("compareOptions", () => new CultureInfo("tr-TR").CompareInfo.GetStringComparer((CompareOptions)0xFFFF));
```

Comment:
Shouldn't the argument name be renamed in the implementation instead? The argument name in both the public APIs is just `options`.

Reply:
https://github.com/dotnet/coreclr/pull/16459#pullrequestreview-98047582